library/common/jni/BUILD

```diff
@@ -36,8 +37,64 @@ cc_binary(
     ],
     linkshared = True,
     deps = [
+        ":android_log",
         ":jni_utility_lib",
         "//library/common:envoy_main_interface_lib",
         "//library/common/api:c_types",
     ],
 )
+
+# Java JNI library
+cc_library(
+    name = "envoy_java_jni_lib",
```

Comment:
nit: let's drop the "envoy_" prefix to align better with other internal targets.